tests/test.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import cv2
+import sys
+import numpy as np
+
+PYTRACKING_PATH = "/home/jetson/ros/pytracking/"
+
+VIDEO_PATH = PYTRACKING_PATH + "/tests/video.mp4"
+sys.path.append(PYTRACKING_PATH)
+
+
+from object_tracker.object_tracker import ObjectTracker
+
+
+@pytest.mark.parametrize(
+    "video_path, run_optical_flow, use_orb",
+    [(VIDEO_PATH, False, False), (VIDEO_PATH, True, False), (VIDEO_PATH, True, True)],
+)
+def test_run_on_frame(video_path, run_optical_flow, use_orb, num_frame_run=20):
```

Comment:
I suggest dividing it into several tests:
1. Simulate with a regular video.
2. Simulate failed OF (using ORB instead). 
3. Simulate with video with randomly black frames. 
4. Simulate with video with randomly random frames instead of images (or very noise frames once in a while).

Reply:
done, except black frames which i not think its it neccessay